web-console/src/singletons/api.ts

```diff
@@ -18,4 +18,9 @@
 
 import axios from 'axios';
 
-export const Api = axios.create();
+function initialize() {
```

Comment:
I meant that instead of setting `axios.defaults` (above) you can pass the instance cnfiguration to the initialize method. So the initialize method would take `{ baseURL, headers }` etc as an argument.